File: mylist(线性表)/student.cpp
#include "student.h"

void student::print()
{
	cout << this->_name;
	cout << this->_age;
	cout << this->_ID;
	cout << this->fraction;
}

void student::getData(string name, string age, string id, string fraction)
{
	this->_name = name;
	this->_age = age;
	this->_ID = id;
	this->fraction = fraction;
}

File: IO2/src/IOwork5/student.java
package IOwork5;

import java.io.Serial;
import java.io.Serializable;

public class student implements Serializable {
    @Serial
    private static final long serialVersionUID = -4392406017290377670L;
    private int age;
    private String name;

    public student(int _age,String _name){
        this.age = _age;
        this.name = _name;
    }

    public String toString(){
        return "student{name = "+name+", age = "+age+"}";
    }
}
